src/System.Drawing.Primitives/src/System/Drawing/Color.cs

```diff
@@ -580,8 +579,7 @@ public override int GetHashCode()
             if (name != null & !IsKnownColor)
                 return name.GetHashCode();
 
-            return HashHelpers.Combine(
-                HashHelpers.Combine(value.GetHashCode(), state.GetHashCode()), knownColor.GetHashCode());
+            return HashCode.Combine(value.GetHashCode(), state.GetHashCode(), knownColor.GetHashCode());
```

Comment:
Why do we still need .GetHashCode? perf?

Reply:
To avoid the unnecessary extra generic instantiations.